docusaurus/docs/iOS/client/llc-client-overview.md

```diff
@@ -13,6 +13,8 @@ The `StreamChat` framework comes with controllers and delegates that you can use
 
 Each controller exposes API functionality and supports delegation. Controllers and their delegates are documented here based on the kind of data they control and allow you to observe. You can find examples on how to build your own view as well.
 
+In addition to controllers, we provide a structured concurrency supported [state layer](state-layer/state-layer-overview.md) with async-await and observable state objects.
```

Comment:
probably modern state layer is better?

Reply:
Sure, we can go with that. It was slightly difficult to name it.